src/blog/microservices-from-development-to-production.md

```diff
@@ -8,6 +8,10 @@ summary: >
 
 ---
 
+<div class="callout warning">
+    Using Kubernetes or DC/OS? Try <a href="https://developer.lightbend.com/docs/lightbend-orchestration/latest/">Lightbend Orchestration</a> for easy deployment of your Reactive Platform services.
```

Comment:
Feedback wanted on the specific text here.

Reply:
I guess the two sentences imply that Lightbend Orchestration works with K and DC/OS, but I think it would be better to state it more clearly. Something like the intro to the Orchestration docs:

See how Lightbend Orchestration simplifies deployment of Lagom microservices (or Reactive Platform Services) with Kubernetes or DC/OS. This commercial suite of tools includes a plugin for creating Docker images, a command line for generating Kubernetes and DC/OS resource and configuration files, and more.

I would be more inclined to check it out with a hint of what it provides, and I'd want to know it was commercial. Also, by saying "and more", this banner won't go stale if more features get added in the future. WDYT?

Reply:
@rstento it's OSS. And not commercial.

Reply:
Changed, thanks!